documentation/slate/source/includes/apis.md

```diff
@@ -166,6 +166,7 @@ Very simple service: you give Navitia some coordinates, it answers you
 | url | Result |
 |----------------------------------------------|-------------------------------------|
 | `coords/{lon;lat}`                           | Detailed address point              |
+| `coverage/{lon;lat}/coords/{lon;lat}`        | Detailed address point              |
```

Comment:
Can you add that it guesses the coverage from coord provided? (like what's done in other APIs)

Reply:
Like that one: https://github.com/CanalTP/navitia/blame/dev/documentation/slate/source/includes/apis.md#L1631